hkmc2/shared/src/main/scala/hkmc2/semantics/Elaborator.scala

```diff
@@ -585,7 +586,10 @@ extends Importer:
       )
     case tree @ Tup(fields) =>
       Term.Tup(fields.map(fld(_)))(tree)
-    case New(body, rfto) => // TODO handle Under
+    // case New(c, rfto) =>
+    //   assert(rfto.isEmpty)
+    //   Term.New(cls(subterm(c), inAppPrefix = inAppPrefix), params.map(subterm(_)), bodo).withLocOf(tree)
+    case ProperNew(body, rfto) => // TODO handle Under
```

Comment:
```suggestion
    case ProperNew(body, rfto) => // TODO handle Under
```

Reply:
Actually I think this is good now:
```suggestion
    case ProperNew(body, rfto) =>
```